manifests/i/Insomnia/InsomniaDesigner/core@2021.7.1/Insomnia.InsomniaDesigner.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created with YamlCreate.ps1 v2.0.2 $debug=QUSU.7-2-0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Insomnia.InsomniaDesigner
+PackageVersion: core@2021.7.1
+MinimumOSVersion: 10.0.0.0
+InstallerType: exe
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/Kong/insomnia/releases/download/core%402021.7.1/Insomnia.Core-2021.7.1-portable.exe
```

```suggestion
  InstallerUrl: https://github.com/Kong/insomnia/releases/download/core%402021.7.1/Insomnia.Core-2021.7.1-portable.exe
```
This is the wrong url, Designer's last release is a year ago https://github.com/Kong/insomnia/releases/tag/designer%402020.5.2